requirements-minimal.txt

```diff
@@ -6,7 +6,7 @@ django-bootstrap-form
 django-ipware
 django-mathfilters
 django-redis
-django>=1.10,<1.10.999
+django~=1.10
```

Comment:
OK, I only made this change to ask why this syntax wasn't used. It seems less flexible than the explicit syntax anyways.

It looks a little silly to use an arbitrary high version number like `.999` though. What about `django>=1.10<1.11.0`?